packages/utils/src/lib/reports/generate-md-reports-diff-utils.ts

```diff
@@ -0,0 +1,182 @@
+import { InlineText, MarkdownDocument, md } from 'build-md';
+import { ReportsDiff } from '@code-pushup/models';
+import { pluralize, pluralizeToken } from '../formatting';
+import { objectToEntries } from '../transform';
+import { DiffOutcome } from './types';
+
+// to prevent exceeding Markdown comment character limit
+const MAX_ROWS = 100;
+
+export function summarizeUnchanged(
+  token: string,
+  { changed, unchanged }: { changed: unknown[]; unchanged: unknown[] },
+): string {
+  return [
+    changed.length > 0
+      ? pluralizeToken(`other ${token}`, unchanged.length)
+      : `All of ${pluralizeToken(token, unchanged.length)}`,
+    unchanged.length === 1 ? 'is' : 'are',
```

Comment:
Could that be a case in our pluralise function?

Reply:
Our pluralize function only handles nouns, where there is an obvious pattern that can be applied to the vast majority of English nouns. For verbs, it would be way more complicated to handle in general, there are lots of different patterns (some described [here](https://www.yourdictionary.com/articles/singular-plural-verb-chart)). I don't think that generalized logic is worth implementing, this is so far the only case where we need to pluralize the verb as well as the noun.